lib/SimpleSAML/Kernel.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+namespace SimpleSAML;
+
+use Symfony\Bundle\FrameworkBundle\FrameworkBundle;
+use Symfony\Bundle\FrameworkBundle\Kernel\MicroKernelTrait;
+use Symfony\Component\Config\Exception\FileLocatorFileNotFoundException;
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\Config\Loader\LoaderInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+use Symfony\Component\DependencyInjection\Loader\DirectoryLoader;
+use Symfony\Component\HttpKernel\Kernel as BaseKernel;
+use Symfony\Component\Routing\RouteCollectionBuilder;
+
+/**
+ * A class to create the container and handle a given request.
+ */
+class Kernel extends BaseKernel
+{
+    use MicroKernelTrait;
+
+    const VERSION = '1.18.0-DEV';
+    const VERSION_ID = 11800;
+    const MAJOR_VERSION = 1;
+    const MINOR_VERSION = 18;
+    const RELEASE_VERSION = 00;
+    const EXTRA_VERSION = 'DEV';
```

Comment:
I don't believe this is good practise? Shouldn't these values (which is saying something about Symfony, rather than SimpleSAML) be left alone?

Reply:
I think you're right about that, yes